### PR TITLE
[Circletracer] Add a file that analyzes the tensor of the subgraph

### PR DESCRIPTION
**Pull request by CODe1995**
This function takes and returns the name and type of each tensor.

ONE-vscode-DCO-1.0-Signed-off-by: Seunghyun Kang <code.57x53@gmail.com>

### CODE REVIEW

**Review by yesfordev — APPROVED** [review]
LGTM

**Review by donggillee-dev — APPROVED** [review]
LGTM

**Comment by CODe1995** [comment]
from draft #179
from issue #66.

File: src/Circlereader/util/initSubgraphNodes.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import {SubGraph} from '../circle-analysis/circle/sub-graph';
+import {Node} from '../type/types';
+
+export function initSubgraphNodes(subgraph: SubGraph): Array<Node> {
+  let nodesArr: Array<Node> = [];
+
+  for (let tensorIdx = 0; tensorIdx < subgraph.tensorsLength(); tensorIdx++) {
+    let tensor = subgraph.tensors(tensorIdx);
+
+    let hashnode: Node = {name: tensor!.name()!, type: tensor!.shapeArray()!};
```
Comment (llFreetimell) [inline src/Circlereader/util/initSubgraphNodes.ts:26]: Can I ask you why `type` is used for indicating `shape`?

Comment (CODe1995) [reply src/Circlereader/util/initSubgraphNodes.ts:26]: image

The variable refers to the arrangement in the photo.

Comment (llFreetimell) [reply src/Circlereader/util/initSubgraphNodes.ts:26]: Hmm... `type` means `float32`, `int8`, etc.
That is shape...!

Comment (CODe1995) [reply src/Circlereader/util/initSubgraphNodes.ts:26]: There are two types of data displayed in the type variable.
It is a type such as `float32`, `int8` and a `shapeArray` expressed together with the type.
Here we do not yet see a type such as `float32`. Instead, it denotes `shapeArray`.
If you look at the places colored with a highlighter, there is a `shapeArray`, so I put it in the type variable.

Comment (llFreetimell) [reply src/Circlereader/util/initSubgraphNodes.ts:26]: Then type such as `float32, `int8` will be displayed later? :)

Comment (CODe1995) [reply src/Circlereader/util/initSubgraphNodes.ts:26]: Yes. Small parts not added now will be implemented later.

Comment (llFreetimell) [reply src/Circlereader/util/initSubgraphNodes.ts:26]: Then please add TODO comment about it in the source code..!

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by donggillee-dev — APPROVED** [review]
LGTM

**Pull request merged**